README.md

```diff
@@ -59,33 +60,106 @@ Table of Contents
 
 Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
 
-## Setup:
-The following `$SHELL` variables should be defined:
+## Setup
 
-- `$BUILD_HOME`
-- `$DATA_PATH`
-- `$ELOG_PATH`
+The `$SHELL` variable `$ELOG_PATH` should be defined:
+
+```
+export ELOG_PATH=/your/favorite/elog/path
+```
+
+Remove and download the setup script to ensure you have the most up-to-date version:
+
+```
+rm -f setup_gemdaq.sh
+wget https://raw.githubusercontent.com/cms-gem-daq-project/sw_utils/master/scripts/setup_gemdaq.sh
+```
 
 Then execute:
 
 ```
-source $BUILD_HOME/cmsgemos/setup/paths.sh
-source $BUILD_HOME/gem-plotting-tools/setup/paths.sh
+source setup.sh -c <cmsgemos tag> -g <gem-plotting tag> -G <gem-plotting dev version optional> -v <vfatqc tag> -V <vfatqc dev version optional>
 ```
 
-If this is the first time you are executing the above commands on `lxplus` it will create a `python v2.7` `virtualenv` for you.  It may take some time to download the necessary packages so be patient and do not interrupt the installation.  To disable this python env execute:
+If a development version is not to be used (normal case), you can drop the options with capital letters (`-G`, `-V`). If this is the first time you are executing the above command, it will create a Python `virtualenv` for you and install the `cmsgemos`, `gemplotting` and `vfatqc` packages. It may take some time to download them, so be patient and do not interrupt the installation.
```

Comment:
Please take a look at https://github.com/cms-gem-daq-project/gem-plotting-tools/pull/109#discussion_r193454753.

I would like specifically this type of system inserted so the user has:

1. The location of where to see the tags, and
2. An explicit understanding of the tag and dev versioning and how that relates.

Reply:
1. Oops, didn't notice the links
2. Can you be more specific? Do you mean an understanding of the whole versionning system?

Reply:
It’s more that you have not shown what is requested. Please copy/paste exactly what I wrote, which I think will give a descriptive generalization, in the now “outdated” comment in the location where it was originally referenced.  Thanks!